core/templates/directives/focus-on.directive.ts

```diff
@@ -16,10 +16,55 @@
  * @fileoverview FocusOn Directive (not associated with reusable
  * components.)
  * NB: Reusable component directives should go in the components/ folder.
+ *
+ * There are two attribute directives in this file. One for AngularJS code and
```

Comment:
This seems fine to me. Just add "NOTE TO DEVELOPERS:" at the beginning of it so as to earmark it clearly for developers, since we want to make sure that if they update one of these, they also update the other.

Reply:
Done!